Project 4_KTs

```diff
@@ -0,0 +1,446 @@
+
+There is one other option for those desiring a different sort of challenge. If you provide alongside your SPARQL submission a translation of the same problem into SQL, complete with documentations, solution, etc. then you may receive half points extra at that kata level (rounded up). For example, if you submit a SPARQL problem that is kata rank 1 and also submit a SQL version of that same problem, you  will receive 35+18=53 points.
+
+---
+
+(1) Level-8 Kata (goal: 0 points/100 points; required.)
+
+Title: Retrieve the name of a city  
+
+Description:
+
+You are new to SPARQL, welcome! To get you started, this kata will ask that you retrieve a specific piece of data by writing a SPARQL query. Write a SPARQL query that retrieves the name of a city. You will be given the URI of a city, and your task is to extract its name using the DBpedia ontology.
+
+Example:
+
+Suppose you are given the URI of Paris: http://dbpedia.org/resource/Paris. Your task is to write a SPARQL query that retrieves the name of this city.
+
+
+Answer:
+
+---
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+SELECT ?cityLabel
+WHERE {
+  <http://dbpedia.org/resource/Paris> rdfs:label ?cityLabel .
+  FILTER (lang(?cityLabel) = 'en')
+}
+---
+
+Return on endpoint:
+
+cityLabel
+-------------
+"Paris"@en
+
+
+Explanation:
+
+The first line defines the prefix for the RDF Schema namespace.
+The SELECT statement retrieves the label of the given city using the WHERE clause.
+The WHERE clause specifies the city URI as the subject, and rdfs:label as the predicate.
+The FILTER clause restricts the results to the English language labels.
+
+
+
+(2) Level-4 kata (10 points/ 100 points)
+
+
+Title: Retrieve the titles of books authored by Jorge Luis Borges
+
+Description:
+
+Your roommate just read The Aleph by Borges and doesn't shut up about it. They expressed interest in reading more works by this author, but lament tha they are too busy this week to research more titles. You want to surprise your roomie with a list of titles by their new favorite author, but instead of researching and finding the titles one by one you figure you should use your newly-gained knowledge of SPARQL. Write a SPARQL query that retrieves the titles of books authored by Jorge Luis Borges. You will be given the URI of Borges, and your task is to extract the titles of the books he authored using the DBpedia ontology.
+
+Example:
+
+Suppose you are given the URI of Borges: http://dbpedia.org/resource/Jorge_Luis_Borges. Your task is to write a SPARQL query that retrieves the titles of books he authored.
+
+Solution to SPARQL query:
+
+---
+PREFIX dbo: <http://dbpedia.org/ontology/>
+
+SELECT ?bookTitle
+WHERE {
+  <http://dbpedia.org/resource/Jorge_Luis_Borges> dbo:notableWork ?book .
+  ?book dbo:author <http://dbpedia.org/resource/Jorge_Luis_Borges> .
+  ?book dbo:name ?bookTitle .
+}
+ORDER BY ?bookTitle
+---
+
```

Comment:
This seems to be a Kata-7, at most Kata-6.

A select query is one of the first functions learned and the triples here involved are entry level.

Kata 4's are hard, but not expert nor black-belt. To make this a Kata-4, I would expect a conceptually challenging task, such as asserting a new triple, (CONSTRUCT, INSERT, CONCAT, or the like), putting some limits that require laser-like precision, or some unavoidably complicated syntax. E.g., a filter for a string or a negation that could easily violate open-world assumption, or to relate the works under a new heading given some shared attribute that is hard to string together.

Doesn't 'ORDER BY' take an argument like 'ASC' or 'DESC'?